src/main/java/bio/terra/cloudfiletodatastore/deltalayer/BigQueryWriter.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cloudfiletodatastore.deltalayer;
+
+import bio.terra.cloudfiletodatastore.deltalayer.model.BqInserts;
+import com.google.cloud.bigquery.*;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class BigQueryWriter {
```

Comment:
This code, along with `DeltaLayerBqGenerator`, `DeltaLayerMessageProcessor` as well as `deltalayer/model/*.java` are `Delta Layer`-centric, would it be possible to include an additional member from `App Services` to review them?